ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -750,6 +750,10 @@ public boolean onEditorAction(TextView v, int actionId, KeyEvent keyEvent) {
                       editText.getText().toString()));
             }
 
+            if (actionId == EditorInfo.IME_NULL) {
+              return !editText.getBlurOnSubmit();
```

Comment:
What is `IME_NULL` and why call `getBlurOnSubmit`? Can you please add a comment explaining what this does?